docs/howto/smart_contract_upload.md

```diff
@@ -0,0 +1,138 @@
+# Smart Contract Upload
```

Comment:
Please reword to "Uploading a Smart Contract" (to be consistent with other topics).